tests/methods/df-dsrg-mrpt2-7-localized-actv/input.dat

```diff
@@ -58,36 +58,22 @@ set forte{
 Ecas, wfn = energy('forte', ref_wfn=wfn, return_wfn=True)
 compare_values(r_avg_cas, Ecas, 8, "SA-CASSCF energy from FORTE CASSCF")
 
-#set forte{
-#  active_space_solver          cas
-#  frozen_docc         [4]
-#  restricted_docc     [9]
-#  active              [4]
-#  avg_state           [[0,1,5]]
-#  rotate_mos          []
-# # fcimo_localize_actv true
-#}
-#Ecas, wfn = energy('forte', ref_wfn=wfn, return_wfn=True)
-#compare_values(r_avg_cas, Ecas, 8, "SA-CASSCF energy from FORTE FCIMO")
-#cubeprop(wfn)
-
 set forte{
-  job_type newdriver
+  job_type            newdriver
```

Comment:
Is ``newdriver`` needed here?

Reply:
Yes, because the job_type is set by the previous block.